core/src/main/scala/chisel3/internal/Error.scala

```diff
@@ -235,7 +235,6 @@ private[chisel3] class ErrorLog {
 
     if (!allErrors.isEmpty) {
       throw new ChiselException("Fatal errors during hardware elaboration. Look above for error list.")
-        with scala.util.control.NoStackTrace
```

Comment:
Chisel have `PrintFullStackTraceAnnotation`. User can use that to print full stacks.

Reply:
I tried PrintFullStackTraceAnnotation, didn't work.

Reply:
I've asked some questions in https://groups.google.com/g/chisel-users/c/61vkD3EfleA

Resulting in a github issue filed [#501 ](https://github.com/ucb-bar/chiseltest/pull/501)

Reply:
have you tried: https://github.com/ucb-bar/chiseltest/pull/501?

Reply:
yes, doesnt work.

the error can probably be reproduced by

0.U(1.W)(0,1)

i will gove a go at creating a test case
...

Reply:
> have you tried: [ucb-bar/chiseltest#501](https://github.com/ucb-bar/chiseltest/pull/501)?

Yes, Updated #501 with a test-case.